misc/python/materialize/parallel_workload/executor.py

```diff
@@ -142,7 +142,7 @@ def execute(
                         raise QueryError(str(e), query)
                 else:
                     try:
-                        if query == "commit":
+                        if query == "commit;":
```

Comment:
Will it always be this string or should we just check both or work with a `startswith`?

Reply:
This entire logic should be reworked. I couldn't come up with a quick solution though. startswith would work for current state of code.